Parsers/ASimNetworkSession/Tests/VMwareCarbonBlackCloud_vimNetworkSession_SchemaTest.csv

```diff
@@ -0,0 +1,110 @@
+﻿Result
+"(1) Warning: Missing recommended field [ASimMatchingHostname]"
+"(1) Warning: Missing recommended field [DstDomain]"
+"(1) Warning: Missing recommended field [DstHostname]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [EventSeverity]"
+"(1) Warning: Missing recommended field [SrcHostname]"
```

Comment:
Done. As suggested [here](https://github.com/Azure/Azure-Sentinel/pull/8774#discussion_r1348924711), we have used coalesce of DvcHostname & SrcIpAddr for SrcHostname.

Reply:
Added condition for DstHostname.
FYI we don't have an Outbound condition so we have done SrcHostname=SrcIpAddr only and not mapped DvcHostname.

Reply:
@jayeshprajapaticrest @niralishah-crest 
why we are mapping hostname with IP?
Instead can we do SrcHostname = DvcHostname when netconn_inbound_b is not true?

Reply:
As you mentioned [here](https://github.com/Azure/Azure-Sentinel/pull/8774#discussion_r1367710150), we can set Dvc fields as Src fields when NetworkDirection is Outbound but in our case we don't have specific condition for Outbound i.e. we can't say directly that netconn_inbound_b is not true means outbound connection. It can be local or remote too. Hence we have removed that mapping and instead mapped SrcHostname with SrcIpAddr as mentioned here
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/21afdee3-45d7-4dd3-84ff-f90b5b2a5412)